code/src/index.css

```diff
@@ -1,13 +1,105 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: 'Trispace', sans-serif;
+  font-size: 1.5vw;    
+  font-weight: 300;  
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+main {
+  width: 80%;
 }
+
+.message-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80%;
+  margin: 20px;
+}
+
+.form, .messages {
+  background: #F2F0F0; 
+  border: 1px solid #000;
+  padding: 10px;
+  box-shadow: 7px 7px #000; 
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+}
+
+.label {
+  line-height: 3em;
+}
+
+.new-message {
+  resize: none;
+  line-height: 1.5em;
+}
+
+.submit-button {
+  width: 50%;
+  padding: 8px;
+  margin: 10px;
+  border: none;
+  border-radius: 20px;
+  background: #f09d9d;
+  font-family: "Trispace"; 
+  font-size: 1.5vw;
+}
+
+.heart-icon {
+  padding: 5px;
+}
+
+.messages {
+  background: none;
+  margin: 20px;
+  display: flex;
+  overflow-wrap: break-word;
+}
+
+.info-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.button-container {
+  margin: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.like-button {
+  width: 40px;
+  height: 40px;
+  border: none;
+  border-radius: 50px;
+  display: flex;
+  align-items: center;
+  background: #EAEAEA;
+} 
+
+.like-button-clicked  {
+  background: #f09d9d;
+}
+
+.like-icon {
+  margin: 5px;
+}
+
+.posted {
+  width: 90%;
+  margin: 5px;
+  display: flex;
+  justify-content: space-between;
+  color: #B4B4B4;
+}
+
+
+
+  
```

Comment:
Great job with the css Emma 💯  Maybe add some media queries as well so it looks amazing on all devices.